### DISCUSSION TITLE
thanhle.blog/ban-so-dieu-gi

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#70] [Ideas]
# thanhle.blog/ban-so-dieu-gi

Để hiểu bản thân hơn và có cách làm việc hiệu quả hơn

https://thanhle.blog/blog/ban-so-dieu-gi

**Comment by toanbku** [comment]
bài viết rất hay và ý nghĩa, cảm ơn anh Thành <3 em đọc như thấy bản thân mình trong đó

**Reply by thanhlmm** [reply]
chúc em bớt sợ 2 năm sắp tới :3

**Comment by Kennn-dev** [comment]
em cũng sợ nhiều thứ như trên, có khi 1 số nỗi sợ nó tái phát nữa 😭, hi vọng anh có vài bài viết nho nhỏ liên quan tới mấy nỗi sợ và cách anh vượt qua nó cho tụi em học hỏi eehhee :3

**Reply by thanhlmm** [reply]
Anh nhiều lắm, viết ra chắc cả sớ :))

Còn nếu em muốn tìm hiểu thêm thì anh suggest đọc thử cuốn này nhé **Search Inside Yourself**

**Comment by phiphuc** [comment]
Bài viết hay quá, em vừa đọc bài này vừa sợ hãi luôn :d
